### PR TITLE
Fidelis - contrib branch

### PR DESCRIPTION
**Pull request by securesynapse**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 1]
```diff
@@ -177,6 +186,346 @@ def generate_time_settings(time_frame=None, start_time=None, end_time=None):
...
+    }
+
+    raw_response = manage_alert_assignuser(data)
+    entry_context = {
+        'AssignedUser': assign_user,
+        'ConclusionID': conclusion_id
+    }
+
+    return_outputs("Assigned User: {} to alert with conclusion ID {}".format(assign_user, conclusion_id),
+                   {'Fidelis.Alert(val.ConclusionID && val.ID == obj.ConclusionID)': entry_context}, raw_response)
```
Comment (ronykoz) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]:
```suggestion
                   {'Fidelis.Alert(val.ConclusionID && val.ConclusionID == obj.ConclusionID)': entry_context}, raw_response)
```

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 2]
```diff
@@ -177,6 +186,346 @@ def generate_time_settings(time_frame=None, start_time=None, end_time=None):
...
+        'comment': comment,
+        'labels': None,
+        'rating': None,
+        'status': 'CLOSED',
+        'action': "STATUS",
+    }
+
+    raw_response = manage_alert_closealert(data)
+
+    return_outputs("Closed alert conclusion ID {}".format(conclusion_id), {}, raw_response)
```
Comment (ronykoz) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: @roysagi We might want to add to the context that the conclusion was closed, what do you think?

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 3]
```diff
@@ -177,6 +186,346 @@ def generate_time_settings(time_frame=None, start_time=None, end_time=None):
...
+
+
+@logger
+def get_alert_ef(alert_id):
+    result = http_request('GET', '/j/rest/v1/alert/ef/{}/'.format(alert_id))
+
+    return result
+
+
+# Grabs the text version of the forensic data - There are other options so Demisto may want to enhance further
```
Comment (ronykoz) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: delete please

Comment (roysagi) [reply Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: done

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py (hunk 2 above)
Comment (roysagi) [reply Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: Thing is the Conclusion ID is no the same as the Alert ID - and the Alert ID is not returned by this command and the conclusion ID is not brought back by other commands... so there is no way to connect the context of the existing alert.

**Pull request closed without merging**